app/views/experiences/card/_like.html.erb

```diff
@@ -1,11 +1,11 @@
 <% if user_signed_in? && current_user.liked?(experience, {vote_scope: experience.default_like[:scope]}) %>
   <%= link_to unlike_challenge_experience_stage_theme_experience_path(@challenge, experience.experience_stage, experience.theme, experience, partial: "card/like"), class: 'list-link', method: :put, remote: true do %>
-    <i class='<%= experience.default_like[:liked][:icon] %>'></i> 
```

Comment:
Lets not include this file / change here - doesn't seem there is any changes to this file.